Storage/FirebaseStorageUITests/FUIImageViewCategoryTests.m

```diff
@@ -30,89 +30,86 @@ @interface FUIImageViewCategoryTests : XCTestCase
 @implementation FUIImageViewCategoryTests
 
 - (void)setUp {
-  [super setUp];
```

Comment:
I found, hole of this Test files, using the Two-Width Space for indent. Is this designed to be ? Most of Objective-C code I see or the Objective-C code format, using the Four-Width Space for indent.

Reply:
OK. I found the Google Styleguide using the 2-Space indent. I've fix it.